packages/wonder-blocks-dropdown/src/components/dropdown-core.js

```diff
@@ -788,33 +788,34 @@ class DropdownCore extends React.Component<Props, State> {
             ? openerStyle.getPropertyValue("width")
             : 0;
 
-        // Vertical padding of the dropdown menu + borders
-        const initialHeight = 12;
-
-        const maxDropdownHeight = getDropdownMenuHeight(
-            this.props.items,
-            initialHeight,
-        );
+        const maxDropdownHeight = getDropdownMenuHeight(this.props.items);
 
         return (
             <View
                 // Stop propagation to prevent the mouseup listener on the
                 // document from closing the menu.
                 onMouseUp={this.handleDropdownMouseUp}
-                role={this.props.role}
                 style={[
                     styles.dropdown,
                     light && styles.light,
                     isReferenceHidden && styles.hidden,
-                    generateDropdownMenuStyles(
-                        minDropdownWidth,
-                        maxDropdownHeight,
-                    ),
-
                     dropdownStyle,
                 ]}
+                testId="dropdown-core-container"
             >
-                {listRenderer}
+                {this.props.isFilterable && this.renderSearchField()}
+                <View
```

Comment:
Created this new wrapper to contain the `OptionItems` and separate the search field from that container.

BEFORE: 
```
<div role="listbox">
  <search />
  <option />
  <option />
  ...
```

AFTER
```
<div container>
  <search />
  <div role="listbox">
    <option />
    <option />
    ...
```

As an extra, this is now more semantically correct as we don't include a `textbox` inside a `listbox`.